dags/mozaggregator_prerelease.py

```diff
@@ -146,7 +146,25 @@
     dag=dag,
 )
 
+mozaggregator2bq_extract = gke_command(
+    task_id="mozaggregator2bq_extract",
+    name="mozaggregator2bq_extract",
+    command=["bin/backfill"],
```

Comment:
If I'm understanding this correctly, this does not make the data available in BQ? Or is this being exposed as an external table?

Reply:
No, there's going to have to be one more step in the airflow job to run the load job. This will start populating the parquet dumps however.